CMakeLists.txt

```diff
@@ -187,7 +187,12 @@ set(CPACK_RPM_RELOCATION_PATHS $ENV{DEST_LIB_DIR})
 set(CPACK_ARCHIVE_COMPONENT_INSTALL OFF)
 
 set(CPACK_DEBIAN_PACKAGE_NAME ${IRODS_CLIENT_NAME})
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${IRODS_VERSION}), libc6, libssl1.1")
+
+if (OPENSSL_VERSION VERSION_LESS "3.0.0")
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${IRODS_VERSION}), libc6, libssl1.1")
+else()
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${IRODS_VERSION}), libc6, libssl3")
+endif()
 
 set(CPACK_RPM_PACKAGE_NAME ${IRODS_CLIENT_NAME})
 if (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "centos linux")
```

Comment:
Consider removing this check as well. We "assume" Enterprise Linux for RPM builds in our other repos now: https://github.com/irods/irods/commit/8dd1b287ce1a24474032787ca7996b2a2baab330#diff-5b5b3e80ba4ffcdf6c8af4e0b8b6d640d9ec0284903e2d83846321e7d2a88a02R94-R97

Reply:
What I mean is that the logic captured by this if statement should be the default (i.e. make it the `else` case).

If this is working on the supported platforms for Globus servers then maybe we don't worry about it. I defer.